oozie/src/main/java/org/apache/falcon/oozie/process/SparkProcessWorkflowBuilder.java

```diff
@@ -163,13 +164,16 @@ private void addOutputFeedsAsArgument(List<String> argList, Cluster cluster) thr
             return;
         }
 
-        for(Output output : entity.getOutputs().getOutputs()) {
+        int numOutputFeed = entity.getOutputs().getOutputs().size();
```

Comment:
@pavankumar526  Being I want input first and then output, I am doing reverse insertion. Inserting output first at 0 index and then input at 0 index in the arg List. So with this, final List will appear in order as input, output and user's argument. As number of arguments are not much, so there will not be much impact in List insertion at 0 index.